workshops/20_python_beginners_workshop/tasks/short_tasks/practice_elif.md

```diff
@@ -0,0 +1,32 @@
+# Exercise 3: Conditions
+
+Imagine that you are a teacher in school and you would like to automatize the calculation of grades.
+
+### Instructions:
+
+1. Create a new variable `student_points` and give it a number between `100` and `0`.
+2. Write first `if` condition that will say `student_points > 90`. If the condition is `True`, this will print `A`.
+3. The second condition will start with the new word `elif` (= else if). This condition will say `student_points >= 80 and student_points <= 90`. If the condition is `True`, this will print `B`. Ask coaches for a help.
```

Comment:
tbh here you do not need the `student_points <= 90`` because you are in an `elif`, so provided that the points are inbetween 1-100, though it can be more.

Reply:
deleted 👍